src/arch/xtensa/configs/override/cannonlake_chrome.config

```diff
@@ -1,2 +1,2 @@
 CONFIG_CAVS_LPS=y
-CONFIG_CAVS_LPRO=y
+CAVS_PM_RO=y
```

Comment:
CONFIG_CAVS_PM_RO ? will we delete _CAVS_LPRO? can you help to explain how should we use LPRO, PM_RO and LPS?

Reply:
IIRC LPRO has been enabled as the default clock due to issues with d0ix, namely lacking a mechanism allowing for gating HPRO. This change addresses that, therefore, switches to HPRO as the default clock.
This might however be not the desired action, so I can revert this particular change in the config if required.